src/cnaas_nms/api/device.py

```diff
@@ -257,8 +257,8 @@ def get(self):
                 total_count = instance.total
 
         resp = make_response(json.dumps(empty_result(status='success', data=data)), 200)
-        resp.headers['X-Total-Count'] = total_count
         resp.headers['Content-Type'] = 'application/json'
+        resp.headers = {**resp.headers, **pagination_headers(total_count)}
```

Comment:
Could perhaps be phrased
`resp.headers |= pagination_headers(total_count)`

Reply:
Since we're using python 3.7 I don't think this will work, and we're using {** syntax elsewhere so I think this is fine

Reply:
Oh ok, indeed both `|` and `|=` are 3.9 features.